README.md

```diff
@@ -26,7 +26,22 @@
           "app_category_type": "public.app-category.productivity"
         },
         "linux": {
-          "icon": "resources/linux/<your-project-id>.png"
+          "icon": "resources/linux/<your-project-id>.png",
+          "debian_config": {
+            "section": "Databases",
+            "depends": [
+              "python"
+            ],
+            "suggests": [
+              "libgnome-keyring0"
+            ]
+          },
+          "redhat_config": {
+            "category": "Applications/Databases",
+            "requires": [
+              "libXScrnSaver(x86-64)"
```

Comment:
Example of the error one should get if `libXScrnSaver(x86-64)` requires is not included:

```
[mongouser@mongo-dev-rhel share]$ cd ~/Downloads/
[mongouser@mongo-dev-rhel Downloads]$ sudo yum install mongodb-compass-dev-1.6.0.x86_64.rpm 
Loaded plugins: langpacks, product-id, search-disabled-repos, subscription-
              : manager
Examining mongodb-compass-dev-1.6.0.x86_64.rpm: mongodb-compass-dev-1.6.0-dev.x86_64
Marking mongodb-compass-dev-1.6.0.x86_64.rpm to be installed
Resolving Dependencies
--> Running transaction check
---> Package mongodb-compass-dev.x86_64 0:1.6.0-dev will be installed
--> Finished Dependency Resolution

Dependencies Resolved

================================================================================
 Package             Arch   Version     Repository                         Size
================================================================================
Installing:
 mongodb-compass-dev x86_64 1.6.0-dev   /mongodb-compass-dev-1.6.0.x86_64 236 M

Transaction Summary
================================================================================
Install  1 Package

Total size: 236 M
Installed size: 236 M
Is this ok [y/d/N]: y
Downloading packages:
Running transaction check
Running transaction test
Transaction test succeeded
Running transaction
  Installing : mongodb-compass-dev-1.6.0-dev.x86_64                         1/1 
  Verifying  : mongodb-compass-dev-1.6.0-dev.x86_64                         1/1 

Installed:
  mongodb-compass-dev.x86_64 0:1.6.0-dev                                        

Complete!
[mongouser@mongo-dev-rhel Downloads]$ cd /usr/share/mongodb-compass-dev/
[mongouser@mongo-dev-rhel mongodb-compass-dev]$ ./MongoDB\ Compass\ Dev 
./MongoDB Compass Dev: error while loading shared libraries: libXss.so.1: cannot open shared object file: No such file or directory
```